blockchain/contracts/ElectionFactory.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT 
```

Comment:
@theGeniusDennis Change your indentation from 2 spaces to 4 spaces only for the `ElectionFactory.sol` file.